packages/firefuel/lib/src/firefuel_collection.dart

```diff
@@ -17,6 +17,8 @@ abstract class FirefuelCollection<T extends Serializable>
     return path;
   }
 
+  DocumentId generateDocumentId() => DocumentId(ref.doc().id);
```

Comment:
So far, in the app we have always used `docId` rather than spelling out `Document`. 

```suggestion
  DocumentId generateDocId() => DocumentId(ref.doc().id);
```